addons/mail/models/res_users.py

```diff
@@ -140,6 +140,11 @@ def write(self, vals):
                     mail_values={'email_to': previous_email},
                     suggest_password_reset=False,
                 )
+        if 'notification_type' in vals:
```

Comment:
Ideally should check that it's different than the current value for each user before sending.

Reply:
I'm wondering maybe we should move that field to user setting instead, and use the setting sync instead?

Reply:
ideally yes, but i am not sure about if it is ok to contact two field from different models and display them on a single form? That's what we have in the preference page so I don't think we want a functional change and move the settings into the `Discuss/settings` page

Reply:
or just use a `related` ? if we really want it